general/device.hpp

```diff
@@ -228,26 +239,36 @@ class Device
    static inline bool Allows(unsigned long b_mask)
    { return Get().backends & b_mask; }
 
+   /** @brief Get the current Host MemoryType. This is the MemoryType used by
+       most MFEM classes when allocating memory used on the host.
+   */
+   static inline MemoryType GetHostMemoryType() { return Get().host_mem_type; }
+
+   /** @brief Get the current Host MemoryClass. This is the MemoryClass used
+       by most MFEM host Memory objects. */
+   static inline MemoryClass GetHostMemoryClass() { return Get().host_mem_class; }
+
    /** @brief Get the current Device MemoryType. This is the MemoryType used by
        most MFEM classes when allocating memory to be used with device kernels.
    */
-   static inline MemoryType GetMemoryType() { return Get().mem_type; }
+   static inline MemoryType GetDeviceMemoryType() { return Get().device_mem_type; }
 
    /** @brief Get the current Device MemoryClass. This is the MemoryClass used
        by most MFEM device kernels to access Memory objects. */
-   static inline MemoryClass GetMemoryClass() { return Get().mem_class; }
+   static inline MemoryClass GetDeviceMemoryClass() { return Get().device_mem_class; }
```

Comment:
Interface change: document in CHANGELOG?

Reply:
Updated the CHANGELOG, added back the method with `deprecated` comments.